docker-entrypoint.sh

```
@@ -14,13 +14,16 @@ SKIP_GAME_DOWNLOAD=${SKIP_GAME_DOWNLOAD:-"0"}
 DOWNLOADER=${DOWNLOADER:-"hytale-downloader"}
 AUTH_CACHE_FILE=${AUTH_CACHE_FILE:-".hytale-auth-tokens.json"}
 ASSET_PACK=${ASSET_PACK:-"Assets.zip"}
-LEVERAGE_AHEAD_OF_TIME_CACHE=${LEVERAGE_AHEAD_OF_TIME_CACHE:-"1"}
+LEVERAGE_AHEAD_OF_TIME_CACHE=${LEVERAGE_AHEAD_OF_TIME_CACHE:-"0"}
+AHEAD_OF_TIME_CACHE_PATH=${AOTCACHE_PATH:-"HytaleServer.aot"}
 ACCEPT_EARLY_PLUGINS=${ACCEPT_EARLY_PLUGINS:-"0"}
 SERVER_MEMORY=${SERVER_MEMORY:-"4096"}
+SERVER_JAR_PATH=${SERVER_JAR:-"HytaleServer.jar"}
```

Shellcheck (0.11.0)

[warning] SERVER_JAR_PATH appears unused. Verify use (or export if used externally).

(SC2034)